misc/pytorch_toolkit/miriad_compression_codes/test/test_train.py

```diff
@@ -0,0 +1,187 @@
+import unittest
+import os
+import numpy as np
+import torch
+import torchvision
+from torch.backends import cudnn
+from utils import get_config
+from utils import download_checkpoint, download_data
+from utils.model import Encoder, Decoder, AutoEncoder
+from torch.utils.data import DataLoader
+from utils.data_loader import CustomDatasetPhase1, CustomDatasetPhase2
+from utils.train_utils import validate_model_phase1
+from termcolor import colored
+import json
+import random
+
+from utils.model import AutoEncoder, Decoder
+from utils.evaluators import compare_psnr_batch, compare_ssim_batch
+cudnn.benchmark = True
+
+
+def load_model(eff_flag=False, it_no=0, alpha=1, beta=1, depth=3, width=64, phase=1):
+    if phase == 1:
+        if eff_flag:
+            model = AutoEncoder(
+                (alpha[it_no]*depth).astype(int), (beta[it_no]*width).astype(int))
+        else:
+            model = AutoEncoder(int(alpha*depth), int(beta*width))
+    else:
+        if eff_flag:
+            model = Decoder((alpha[it_no]*depth).astype(int),
+                            (beta[it_no]*width).astype(int))
+        else:
+            model = Decoder(int(alpha*depth), int(beta*width))
+
+    return model
+
+
+# validate_model_phase1(config, test_dataloader, model, msecrit)
+
+
+def create_train_test_for_densenet121():
+    class TrainerTest(unittest.TestCase):
+        @classmethod
+        def setUpClass(cls):
+            config = get_config(action='train')
+            cls.config = config
+            if os.path.exists(config["default_image_path"]):
+                image_path = config["default_image_path"]
+            else:
+                if not os.path.exists(config["image_path"]):
+                    download_data()
+                image_path = config["image_path"]
+
+            dataset_train = RSNADataSet(
```

Comment:
RSNADataSet?

Reply:
It was my mistake Sir. That was the folder that you advised not to use and it was for a separate task. It has been deleted now and the change will be reflected in the branch.